iree/turbine/kernel/wave/compile.py

```diff
@@ -176,4 +187,6 @@ def get_binary_path():
     if options.wave_runtime:
         binary_path = get_binary_path()
 
-    return WaveKernel(options, compiled_wave_vmfb, asm, binary_path)
+    return WaveKernel(
+        options, compiled_wave_vmfb, asm, binary_path, bound_scalar_symbols
+    )
```

Comment:
Do we need to add this to the cache? What do we need to change to caching with this change?

Reply:
It's constructed in `LaunchableWave` init and always available so we don't need to cache it.

Reply:
great